src/bp/nlu-core/intents/exact-intent-classifier.ts

```diff
@@ -32,10 +32,16 @@ export const modelSchema = Joi.object()
   .keys(schemaKeys)
   .required()
 
-export class ExactIntenClassifier implements IntentClassifier {
-  private static _name = 'Exact Intent Classifier'
+export class ExactIntenClassifier implements NoneableIntentClassifier {
```

Comment:
I don't understand why exact intent should ne noneable ?

Reply:
well if there's no exact match => its none

Reply:
this makes sure the sum of all confidence is 1